src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -47,100 +47,111 @@ export default function PreferencesComponent() {
 				successMessage='updated.preferences'
 				failureMessage='failed.to.submit.changes'
 			/>
-			<div style={bottomPaddingStyle}>
+			<h3 className='border-bottom'>{translate('meter.settings')}</h3>
```

Comment:
I thought I had an earlier comment on this but I cannot find it. Thus, I think either the sections should be alphabetical (best if going with side bar) or it should be graph then site then meter (order likely changed). I'll let you choose what you think is best.

Reply:
The setting sections was initially alphabetical, however in your [comment](https://github.com/OpenEnergyDashboard/OED/pull/1324#discussion_r1700553357) you requested a change to move Graph Settings to the bottom. 
I have reverted that change in commit [ec86c72](https://github.com/OpenEnergyDashboard/OED/pull/1324/commits/ec86c727c93cd62fcd423ca20dcab27b5cee11fe) and made the setting sections alphabetical.

Reply:
That was my mistake. I had meant to ask to have the meter settings last (consistent with this comment). However, you chose alphabetical per above and that is fine. I'm hoping we use the sidebar and then this is a good choice.